pkg/extensions/search/common/common_test.go

```diff
@@ -2609,6 +2629,198 @@ func TestSearchSize(t *testing.T) {
 	})
 }
 
+func TestImageSummary(t *testing.T) {
+	port := GetFreePort()
```

Comment:
Please move this code under `Convey("GraphQL query ImageSummary", t, func() {`
There's a strange behavior we found today regarding this test in combination with the RepoDB related work: we can't reuse the controller object to start/stop the server multiple times.
Should not reproduce without that work but it may cause issues in the future when we merge these.